src/JSONdata/genres.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "genre_id" : 91,
+    "name" : "Habeeb",
+    "Items" : [
+      {
+        "on_spotify" : true,
+        "publish_date" : "2022-09-01"
+      },
+      {
+        "on_spotify" : false,
+        "publish_date" : "2023-02-01"
+      }
+    ]
+  },
+  {
+    "genre_id" : 78,
+    "name" : "Clasic",
+    "Items" : [
+      {
+        "on_spotify" : true,
+        "publish_date" : "2034-06-06"
+      }
+    ]
+  },
+  {
+    "genre_id" : 77,
+    "name" : "Promise",
+    "Items" : [
+      {
+        "on_spotify" : false,
+        "publish_date" : "2309-03-03"
+      }
+    ]
+  }
+]
```

Comment:
There already exists a file folder in the src sub directory which should handle saving all of the json files. to keep the code base  clean and not have duplicate folders for a single task i suggest you add the whole content of this json file into the files folder.